service/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateSentinelAlertRulesStep.java

```diff
@@ -137,4 +139,9 @@ by AccountName, CallerIpAddress, Identity, bin(TimeGenerated, 10m)
     alertRulesHelper.createAlertRule(
         fileAccessAttemptsRule, "UnauthorizedFileAccessAttempts", mrgName, workspaceName);
   }
+
+  @Override
+  public List<ResourceNameRequirements> getResourceNameRequirements() {
```

Comment:
Agree. That might be an option too. But current implementation enforces new steps to implement it instead of just having default implementation by default and later adjust it since default implementation is an exception here.